modules/util/css-builder/src/main/java/com/liferay/css/builder/CSSBuilder.java

```diff
@@ -58,67 +65,56 @@
  * @author Shuyang Zhou
  * @author David Truong
  */
-public class CSSBuilder implements AutoCloseable {
+public class CSSBuilder implements AutoCloseable, IDefaultProvider {
```

Comment:
We never used this `IDefaultProvider` interface before, are you sure it's needed?